apps/pdc-frontend/README.md

```diff
@@ -1 +1,12 @@
 # Utrecht PDC Frontend
+
+## Open Forms Environment configuration
+
+If you intend on testing or developing embedded Open Forms, make sure you configure the following variable:
+
+`OPEN_FORMS_API_URL=...` e.g. `http://localhost:8000/api/v2/`
+
+Optionally you can define the following variables, which is useful if you use a local SDK version on which you are developing.
+
+`OPEN_FORMS_SDK_URL=...` e.g. `http://localhost:8000/static/sdk/open-forms-sdk.js`
+`OPEN_FORMS_CSS_URL=...` e.g. `http://localhost:8000/static/sdk/open-forms-sdk.css`
```

Comment:
Ik zou deze info naar de README in de root verplaatsen, anders worden ze niet gespot door devops.